wallet-new/server/Cardano/Wallet/API/V1/Swagger/Example.hs

```diff
@@ -11,45 +18,64 @@ class Arbitrary a => Example a where
     example :: Gen a
     example = arbitrary
 
-instance {-# OVERLAPPABLE #-} Arbitrary a => Example a
+-- | Any @Example@ instance that we want to ovveride must be matched on
+-- the type level in this definition here.
+type family (F a) :: Bool where
```

Comment:
Really good trick https://kseo.github.io/posts/2017-02-05-avoid-overlapping-instances-with-closed-type-families.html in the article. Ty for pointing it out